setup.py

```diff
@@ -53,26 +53,26 @@
     'google-cloud-bigquery >= 0.25.0, < 0.26dev',
     'google-cloud-bigtable >= 0.25.0, < 0.26dev',
     'google-cloud-core >= 0.25.0, < 0.26dev',
-    'google-cloud-datastore >= 1.1.0, < 2.0dev',
+    'google-cloud-datastore >= 1.1.0, < 1.2dev',
     'google-cloud-dns >= 0.25.0, < 0.26dev',
-    'google-cloud-error-reporting >= 0.25.0, < 0.26dev',
+    'google-cloud-error-reporting >= 0.25.1, < 0.26dev',
     'google-cloud-language >= 0.25.0, < 0.26dev',
-    'google-cloud-logging >= 1.1.0, < 2.0dev',
+    'google-cloud-logging >= 1.1.0, < 1.2dev',
     'google-cloud-monitoring >= 0.25.0, < 0.26dev',
     'google-cloud-pubsub >= 0.26.0, < 0.27dev',
     'google-cloud-resource-manager >= 0.25.0, < 0.26dev',
     'google-cloud-runtimeconfig >= 0.25.0, < 0.26dev',
     'google-cloud-spanner >= 0.25.0, < 0.26dev',
     'google-cloud-speech >= 0.26.0, < 0.27dev',
-    'google-cloud-storage >= 1.2.0, < 2.0dev',
+    'google-cloud-storage >= 1.2.0, < 1.3dev',
```

Comment:
The issue is "us". We have made every other package in here depend explicitly on the `0.25.micro` series of `google-cloud-core`. If one of the (possibly many) future releases of `google-cloud-storage`  in the `1.minor.micro` series depends on a **different** segment of `google-cloud-core`, then installing the umbrella package will lead to an incompatible set of version constraints.